packages/react-native-oursky/index.d.ts

```diff
@@ -89,6 +89,7 @@ export interface CountryPickerProps extends ExtraTextProps {
   selectedValue?: string;
   backButtonText?: React.ReactNode;
   headerTitle?: React.ReactNode;
+  defaultByDeviceCountry?: boolean;
```

Comment:
https://github.com/oursky/react-native-oursky/pull/13#discussion_r239316721

Please make it non-optional.